pixi.toml

```diff
@@ -0,0 +1,50 @@
+[project]
+name = "multiregex"
+description = "Quickly match many regexes against a string. Provides 2-10x speedups over naïve regex matching."
+channels = ["conda-forge"]
+platforms = ["osx-arm64", "osx-64", "linux-64", "win-64"]
+
+[tasks]
+postinstall = "pip install --no-build-isolation --no-deps --disable-pip-version-check -e ."
+
+[dependencies]
+python = ">=3.8"
```

Comment:
Can we try >=3.6 and exclude osx-arm64 from older python versions?

Reply:
How to do this with pixi?

Reply:
I will not support python versions which reached EOL. Old versions of this package are available, but there will be no support going forward.